prow/config.yaml

```diff
@@ -170,6 +170,11 @@ presubmits:
         env:
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /etc/service-account/service-account.json
+        - name: COVERALLS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: k8s-multicluster-ingress-coveralls-token
+              key: coveralls
```

Comment:
(unfortunately we don't use upstream types.go, ref https://github.com/kubernetes/test-infra/blob/master/prow/kube/types.go#L170-L172, you won't get your yaml parsed properly)
/shrug

Reply:
gah. 

It shouldn't harm anything in test-infra but we should support this xref: #1986 :-(

Reply:
cc @cjwagner @kargakis what do you think about taking another look at upstreaming these types?

Reply:
By "upstreaming" you mean "update our types in prow", right? SGTM if so, eventually we should fix #1986.

Reply:
okay, so given that this doesn't work now, I assume I should go back to mounting the secret as a file?

Reply:
> By "upstreaming" you mean "update our types in prow", right? SGTM if so, eventually we should fix #1986.

Yes let's fix that.

> okay, so given that this doesn't work now, I assume I should go back to mounting the secret as a file?

You can, but you'll need to tweak your makefile to read the file in and pass it to goveralls.

Reply:
okay, I've updated this PR and the Makefile PR. PTAL.